### ISSUE TITLE
Disable GitHub review requirement now that we rely on prow

### ISSUE THREAD

**Issue by jlewi** [kubeflow/community#22]
We are moving towards using OWNERS and Prow to ensure CLs are LGTM'd and approved.

Should we disable for our repositories the GitHub check that PRs are approved? It seems like that just forces users to /lgtm/approve requests twice.

**Comment by gaocegege** [comment]
SGTM

**Comment by ddysher** [comment]
@jlewi with prow, we don't need github review requirement, but we want to make sure prow works well for us at first.

**Comment by jlewi** [comment]
@ddysher Agreed. I think the GitHub review status checks block auto-merge. So if we keep them on people need to approve twice (once with GitHub, once with prow) and possible a third time if Reviewable is enabled as a status check. 

So my inclination is to start disabling the GitHub status checks so we can really experiment with the Prow workflow. We can always reenable them if need be.

But now that you convinced me to try auto-merge, I think its fantastic. I think it will significantly reduce the toil around managing PRs.

**Comment by gaocegege** [comment]
I think it is time to disable GitHub review feature now, since the bot works well in tf-operator
